ChangeLog.md

```diff
@@ -90,7 +90,7 @@ Bug fixes:
 * Fix `subdirs` for git repos in `extra-deps` to match whole directory names.
   Also fixes for `subdirs: .`. See
   [#4292](https://github.com/commercialhaskell/stack/issues/4292)
-
```

Comment:
Please keep the extra newline following the list, I believe it's needed by tooling (but not quite sure tbh).